ntbs-service/Pages/Index.cshtml.cs

```diff
@@ -9,21 +8,23 @@ namespace ntbs_service.Pages
 {
     public class IndexModel : PageModel
     {
-        private readonly INotificationService service;
+        private readonly INotificationService Service;
+        private readonly IUserService TbServicesManager;
```

Comment:
Think it would be clearer if these were renamed to notificationService and userService locally.

Reply:
yeah, good catch